hpOneView/resources/servers/server_profiles.py

```diff
@@ -399,8 +295,8 @@ def get_available_storage_systems(self, start=0, count=-1, filter='', sort='', *
         Returns:
             list: Available storage systems.
         """
-        uri = self.__build_uri_with_query_string(kwargs, '/available-storage-systems')
-        return self._client.get_all(start=start, count=count, filter=filter, sort=sort, uri=uri)
+        uri = self._helper.build_uri_with_query_string(kwargs, '/available-storage-systems')
+        return self._helper.get_all(start=start, count=count, filter=filter, sort=sort, uri=uri)
 
     def get_available_targets(self, **kwargs):
         """
```

Comment:
we need to add the filter query param in the comments

Reply:
Added